oci/pkg/oci/image.go

```diff
@@ -0,0 +1,165 @@
+package oci
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/containerd/containerd/remotes/docker"
+	orascontent "github.com/deislabs/oras/pkg/content"
+	orascontext "github.com/deislabs/oras/pkg/context"
+	"github.com/deislabs/oras/pkg/oras"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func init() {
+	orascontext.GetLogger(context.Background()).Logger.SetLevel(logrus.ErrorLevel)
```

Comment:
If we expect/hope this code will end up in either tkn or Tekton cobtrollers we should try to standardize on the logging used in those codebases, I don't think they use logrus today.

Reply:
I believe this is actually just here to suppress the built-in logger of the `oras` package which spits out warnings for using a custom mime that isn't publicly registered anywhere. I don't think this PR specifies a logger and is, unfortunately, only importing logrus to set the log level.